parsers/html.c

```diff
@@ -346,8 +346,8 @@ static bool skipScriptContent (tokenInfo *token, long *line, long *lineOffset)
 	bool found_start = false;
 	bool found_script = false;
 
-	long line_tmp[2];
-	long lineOffset_tmp[2];
+	unsigned long line_tmp[2];
+	unsigned long lineOffset_tmp[2];
```

Comment:
Oh, this is mistake. This doesn't suppress the original warning. The original warnings are:
```
parsers/html.c:439:6: warning: ‘lineOffset_tmp[1]’ may be used uninitialized in this function [-Wmaybe-uninitialized]
      makePromise ("JavaScript", startLineNumber, startLineOffset,
      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
          endLineNumber, endLineOffset, startSourceLineNumber);
          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
parsers/html.c:439:6: warning: ‘line_tmp[1]’ may be used uninitialized in this function [-Wmaybe-uninitialized]
```

I will remove this commit and add a fix for this warning.